ui.apps/src/test/components/commerce/product/addToCartTest.js

```diff
@@ -112,7 +112,11 @@ describe('Product', () => {
 
         it('adds a product to the cart on click', () => {
```

Comment:
This test should check that if you click on the button, that `window.CIF.MiniCart.addItem` was called. You can remove the spy on the `_onAddToCart` method.

Reply:
done